docs/newbs_getting_started.md

```diff
@@ -70,6 +70,18 @@ You will need to install Git and Python. It's very likely that you already have
 * Fedora / Red Hat / CentOS: `yum install git python3 && python3 -m pip install qmk`
 * Arch: `yay -S qmk` (or use any other AUR Helper)
 
+### FreeBSD
```

Comment:
Adding id tags helps people doing translations, and is something we've started to do recently.

```suggestion
### FreeBSD :id=freebsd
```